Dockerfile.dev

```diff
@@ -9,7 +9,9 @@ COPY requirements_production.txt .
 COPY requirements_development.txt .
 RUN pip install -r requirements_development.txt
 
+COPY src/* src/
```

Comment:
I don't think this is necessary since the folder is mounted: https://github.com/OpenSlides/openslides-media-service/blob/b2562beb94d203c16940366ff25380ea833ed07a/docker-compose.test.yml#L20

Reply:
oh it is? I had problems testing the dev image, cause it couldnt find the src/* files
Might have been because i dont have a proper dev environment